src/components/data-management/ProjectsList.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Space } from 'antd';
+
+import ProjectCard from './ProjectCard';
+
+const ProjectsListContainer = (props) => {
+  const { height, filter } = props;
+
+  const projects = useSelector((state) => state.projects);
+
+  return (
+    <Space direction='vertical' style={{ width: '100%', height: height - 90 }}>
+      {
+        projects.ids.map((projectUuid) => {
+          const project = projects[projectUuid];
+
+          const matchFilter = project.name.match(filter)
+              || project.experiments.some((experimentId) => experimentId.match(filter))
+              || projectUuid.match(filter);
+
+          if (!matchFilter) return <></>;
+
+          return (
+            <ProjectCard
+              key={project.uuid}
+              projectUuid={project.uuid}
+            />
+          );
+        })
+      }
+    </Space>
+  );
+};
+
+ProjectsListContainer.propTypes = {
+  height: PropTypes.number,
+  filter: PropTypes.object.isRequired,
```

Comment:
This is not right -- I just checked index.jsx page, you are not passing `filter` in there. Either correct index.jsx, or mark this prop as not required and make sure the code won't fail if it is missing.

Reply:
Actually I worked it out, there are two components called `ProjectsListContainer`, which you will separately need to fix. For here, I would suggest that instead of passing filter as prop, you move `  const [filterParam, setFilterParam] = useState('');` out of the other `ProjectsListContainer` and have it here directly.